draft-ietf-rtcweb-jsep.xml

```diff
@@ -1182,11 +1182,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host ]]></sourcecode>
         <section anchor="sec.createoffer" numbered="true" toc="default">
           <name>createOffer</name>
           <t>The createOffer method generates a blob of SDP that
-          contains an offer per <xref target="RFC3264" format="default"/> with the supported
-          configurations for the session, including descriptions of the
-          media added to this PeerConnection, the codec, RTP, and RTCP
-          options supported by this implementation, and any candidates
-          that have been gathered by the ICE agent. An options
+          contains an offer per <xref target="RFC3264" format="default"/> that reflects
+          the current configuration of the session, including descriptions of the
+          media added to this PeerConnection and the codec, RTP, and RTCP
```

Comment:
```suggestion
          media added to this PeerConnection and the codecs, RTP, and RTCP
```

Reply:
here, "codec" is an adjective modifying "options"